### PR TITLE
Hg38

### PR DESCRIPTION
**Pull request by hannasoderstrom1**
Added lists of contigs for hg38.
Use reference (hg19 or hg38) as argument

### CODE REVIEW

File: alissa_API_tools.py [hunk 1]
```diff
@@ -68,9 +68,10 @@ def alissa_upload(accession, sex, alissa_folder, vcf_path, output_folder, size,
               help='Path to a folder where the VCF will be written if the input VCF is larger than the size argument. In that case, files will be loaded to Alissa from that folder. Default: "/tmp".')
 @click.option('-i', '--production-instance', type=click.Choice(['production', 'test']) , default="test",
              help='What Alissa instance should be used. Default: "test".') #TODO invert the logic once testing is finished.
+@click.option('-ref', '--reference', help='Reference genome (hg19 or hg38). Default: hg19', default = 'hg19')
```
Comment (JPWerling) [inline alissa_API_tools.py:71]: Change this to ` '-ref', help= '', type=click.Choice(['hg19', 'hg38'], case_sensitive=False, default='hg19'))` to ensure no other weird inputs are accepted.

File: chunk_vcf.py [hunk 2]
```diff
@@ -135,9 +143,11 @@ def prepare_and_split_vcf(vcf, outfolder, size, logpath=None):
               help='Path to a folder where VCF will be written if the input VCF is larger than the size argument')
 @click.option('-s', '--size', required=True, type=int,
               help='Size in bytes. If the VCF exceed this size, it will be split into 2, 3 or 4 VCFs')
+@click.option('-ref', '--reference', required=True,
```
Comment (JPWerling) [inline chunk_vcf.py:146]: See previous comment about `type=click.Choice([])`

**Review by JPWerling — APPROVED** [review]
Looks to work as is. Passing the `reference` variable around multiple times is not the best but it's not a major issue.

File: alissa_API_tools.py (hunk 1 above)
Comment (hannasoderstrom1) [reply alissa_API_tools.py]: solved in 58a48a3

File: chunk_vcf.py (hunk 2 above)
Comment (hannasoderstrom1) [reply chunk_vcf.py:146]: solved in 58a48a3

**Pull request merged**